src/services.py

```diff
@@ -1,107 +0,0 @@
-
-    recon_df = pd.DataFrame()
-
-    for _, row in fund_name_and_reporting_date.iterrows():
-        fund_name, reporting_date = row['FUND NAME'], row['REPORTING DATE']
-        logging.info(f"Running EQUITY recon report for fund name: {fund_name}, reporting date: {reporting_date}")
-
-        recon_report_equity_query = get_equity_recon_report_with_fund_name_and_reporting_date_query(fund_name,
-                                                                                                      reporting_date)
-        recon_equity_results = db.fetch_as_df(recon_report_equity_query)
-
-        if not recon_equity_results.empty:
-            logging.warning(f"Require EQUITY reconciliation for fund name: {fund_name}, reporting date: {reporting_date}!")
-            recon_df = pd.concat([recon_df, recon_equity_results], ignore_index=True)
-
-    logging.info("Completed recon report for EQUITY")
-
-    return recon_df.reset_index(drop=True)
-
-def generate_recon_report_for_bonds(db: SQLiteDatabase) -> pd.DataFrame:
-    fund_name_and_reporting_date = db.fetch_as_df(get_distinct_fund_name_and_reporting_date_query())
-
-    recon_df = pd.DataFrame()
-
-    for _, row in fund_name_and_reporting_date.iterrows():
-        fund_name, reporting_date = row['FUND NAME'], row['REPORTING DATE']
-        logging.info(f"Running BONDS recon report for fund name: {fund_name}, reporting date: {reporting_date}")
-
-        recon_report_bonds_query = get_bonds_recon_report_with_fund_name_and_reporting_date_query(fund_name, reporting_date)
-        recon_bonds_results = db.fetch_as_df(recon_report_bonds_query)
-
-        if not recon_bonds_results.empty:
-            logging.warning(f"Require BONDS reconciliation for fund name: {fund_name}, reporting date: {reporting_date}!")
-            recon_df = pd.concat([recon_df, recon_bonds_results], ignore_index=True)
-
-    logging.info("Completed recon report for BONDS")
-
-    return recon_df.reset_index(drop=True)
-
-def generate_recon_report(db: SQLiteDatabase, output_folder: str) -> pd.DataFrame:
-    fund_name_and_reporting_date = db.fetch_as_df(get_distinct_fund_name_and_reporting_date_query())
-
-    recon_df = pd.DataFrame()
-
-    for _, row in fund_name_and_reporting_date.iterrows():
-        fund_name, reporting_date = row['FUND NAME'], row['REPORTING DATE']
-        logging.info(f"Running recon report for fund name: {fund_name}, reporting date: {reporting_date}")
-
-        recon_report_query = get_recon_report_query(fund_name, reporting_date)
-        recon_results = db.fetch_as_df(recon_report_query)
-
-        if not recon_results.empty:
-            logging.warning(f"Require reconciliation for fund name: {fund_name}, reporting date: {reporting_date}!")
-            recon_df = pd.concat([recon_df, recon_results], ignore_index=True)
-
-    logging.info("Completed recon report")
-
-    os.makedirs(output_folder, exist_ok=True)
```

Comment:
As before, `output_folder` and `recon_report_filename` would be better as pydantic attributes, set via the CLI as necessary.